lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -78,6 +78,14 @@ class ReportUIFeatures {
   initFeatures(report) {
     this.json = report;
 
+    // TODO(i18n): Modifying Util singleton is not the best way to i18n this.
+    // Related TODO: https://github.com/GoogleChrome/lighthouse/pull/5713/files#r204948699
+    // Update util UIStrings, just like in report-renderer, we need to do it again b/c they have been reset.
+    Util.cacheUIStrings();
+    if (report.i18n && report.i18n.rendererFormattedStrings) {
```

Comment:
"pass the lhr to everything" (https://github.com/GoogleChrome/lighthouse/pull/5713/files#r204939491) seems good to me. Mostly is just passing the LHR through each of the category renderer constructors. preferably this should be done before this PR lands, as this is getting complicated to reason about.

Reply:
https://github.com/GoogleChrome/lighthouse/pull/10153/